src/wattsi.pas

```diff
@@ -681,11 +686,61 @@    TCrossReferences = record
                            Scratch.Append($002E);
                         Scratch.Append(IntToStr(CurrentSectionNumber[CurrentHeadingRank]));
                      end;
+                     if (Variant = vDEV) then
+                     begin
+                        ID := Element.GetAttribute('id').AsString;
+                        SectionNumber := Scratch.AsString;
+                        HeadingText := DelSpace1(StringReplace(StringReplace(Element.TextContent.AsString, #$0A, ' ', [rfReplaceAll]), #$22, '\' + #$22, [rfReplaceAll]));
```

Comment:
Can you comment on what this line is doing, and/or split it up into multiple lines? I think so far I've deduced it's replacing nbsps with normal spaces, and escaping quote marks. Not sure about the DelSpace1 part, or why the nbsps are there in the first place.